ratis-server/src/test/java/org/apache/ratis/server/impl/RaftConfigurationTestUtil.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.ratis.server.impl;
+
+import org.apache.ratis.protocol.*;
+import org.apache.ratis.server.*;
+
+public class RaftConfigurationTestUtil {
+    public static boolean isHighestPriority(RaftConfiguration config, RaftPeerId peerId) {
+        return ((RaftConfigurationImpl)config).isHighestPriority(peerId);
+    }
```

Comment:
Let's put this in RaftServerTestUtil since there are some other conf related methods.

Reply:
Okay, should be now fixed!